package.json

```diff
@@ -43,13 +43,15 @@
   "license": "MIT",
   "devDependencies": {
     "@tsconfig/recommended": "^1.0.2",
+    "@types/mysql": "^2",
     "dotenv": "^16.0.3",
     "husky": "^8.0.3",
     "lint-staged": "^13.1.1",
     "prettier": "^2.8.3",
     "typescript": "^5.0.0"
   },
   "dependencies": {
+    "mysql": "^2.18.1",
```

Comment:
Shouldn't be a hard dependency here. Please see this guide for the right way to add third-party deps:
https://github.com/hwchase17/langchainjs/blob/main/.github/contributing/INTEGRATIONS.md